src/dataProvider/services/DataProvider.js

```diff
@@ -15,33 +15,28 @@ class DataProvider {
   /**
    * Get media composition by URN.
    *
-   * @param {String} urn urn:rts:video:9800629
-   * @param {Boolean} onlyChapters
+   * @param {String} urn URN of the media composition.
+   * @param {Boolean} [onlyChapters=true] Whether to retrieve only chapters or not.
    *
-   * @returns {Object} media composition json object
+   * @returns {Promise<{mediaComposition: MediaComposition}>} Promise that resolves with the `mediaComposition` object.
+   * @throws {Response} If the response is not ok.
```

Comment:
Isn't the `Response` also wrapped into a `Promise`, as in : 

```suggestion
   * @throws {Promise<Response>} If the response is not ok.
```

Reply:
Thanks! It's fixed.